tide_webform.module

```diff
@@ -162,3 +163,12 @@ function tide_webform_webform_submission_create_access(AccountInterface $account
   }
   return AccessResult::neutral();
 }
+
+/**
+ * Implements hook_ENTITY_TYPE_presave().
+ */
+function vicgovau_core_webform_presave(EntityInterface $entity) {
```

Comment:
Should this be `tide_webform_webform_presave()`?

Reply:
ah....yes. typo issue. thanks @anthony-malkoun